Python Examples/TestMonitor/CreateResultsAndSteps/create_results_and_steps.py

```diff
@@ -118,6 +124,70 @@ def generate_step_data(
     return step_data
 
 
+def is_partial_success_response(response: Dict) -> bool:
+    return "error" in response.keys()
+
+
+def get_test_result() -> Dict:
+    test_result = {
+        "programName": "Power Test",
+        "status": {
+            "statusType": "RUNNING",
+            "statusName": "Running"
+        },
+        "systemId": None,
+        "hostName": None,
+        "properties":None,
+        "serialNumber": str(uuid.uuid4()),
+        "operator": "John Smith",
+        "partNumber": "NI-ABC-123-PWR1",
+        "fileIds":None,
+        "startedAt": str(datetime.datetime.utcnow()),
+        "totalTimeInSeconds": 0.0
+    }
+
+    return test_result
+
+
+def create_result() -> Dict:
+    test_result = get_test_result()
+
+    response = test_data_manager_client.create_results(results=[test_result])
+    if is_partial_success_response(response) :
+        raise Exception("The test result is not created, please check for correct result details and you have correct access for creating the new test results")
```

Comment:
You're not consistent with raising an exception vs printing. Is that because we can't continue without a result, but you think it would be okay to continue if creating a step fails? That isn't true when creating the parent step at least, but I think we should just raise in these error cases and not try to continue. And if we're always raising I think its probably better to put this error handling in the client rather than cluttering up the main example code.

It would also be good to include the error information from the response.

Reply:
raised the exception if there is any error while creating the parent step.